src/sensitivities/array.jl

```diff
@@ -1,3 +1,9 @@
 # Let the user get the `size` and `length` of `Node`s.
 Base.size(x::Node, dims...) = size(x.val, dims...)
 Base.length(x::Node) = length(x.val)
+
+# Sensitivity for the first argument of `reshape`.
+import Base.reshape
+@explicit_intercepts reshape Tuple{∇RealArray, Vararg{Int64}} [true, false]
+Nabla.∇(::typeof(reshape), ::Type{Arg{1}}, _, y, ȳ, A::∇RealArray, args...) =
```

Comment:
You shouldn't need to do `Nabla.` here; just `∇` should suffice.

Reply:
Will also adjust this.